wgpu-core/src/device/resource.rs

```diff
@@ -2612,10 +2612,16 @@ impl Device {
             .map(|bgl| bgl.raw())
             .collect::<ArrayVec<_, { hal::MAX_BIND_GROUPS }>>();
 
+        #[cfg(feature = "indirect-validation")]
+        let additional_flags = hal::PipelineLayoutFlags::INDIRECT_BUILTIN_UPDATE;
+        #[cfg(not(feature = "indirect-validation"))]
+        let additional_flags = hal::PipelineLayoutFlags::empty();
+
```

Comment:
suggestion(non-blocking): This seems like a great time to use `cfg!(…)`:

```suggestion
        let additional_flags = if cfg!(feature = "indirect-validation") {
            hal::PipelineLayoutFlags::INDIRECT_BUILTIN_UPDATE
        } else {
            hal::PipelineLayoutFlags::empty()
        };
```